security-team/SECURITY.md

```diff
@@ -0,0 +1,23 @@
+# Security Policy
+
+## Report a vulnerability
+
+Please follow steps listed [here](report-a-vulnerability.md) to report a vulnerability.
+
+## Security release process
+
+The KubeEdge community will strictly handle the reporting vulnerability according to this [procedure](security-release-process.md).
+
+## Mailing lists
+
+- [cncf-kubeedge-security@lists.cncf.io](mailto:cncf-kubeedge-security@lists.cncf.io), is for reporting security concerns to the KubeEdge Security Team, who uses the list to privately discuss security issues and fixes prior to disclosure.
+
+- [cncf-kubeedge-distrib-announce@lists.cncf.io](mailto:cncf-kubeedge-distrib-announce@lists.cncf.io), is for advance private information and vulnerability disclosure. 
+
+More details of group membership is managed [here](security-groups.md).
+
+See the [private-distributors-list](private-distributors-list.md) for information on how KubeEdge distributors or vendors can apply to join this list.
+
+## Supported versions
+
+See the [KubeEdge releases page](https://github.com/kubeedge/kubeedge/releases) for information on supported versions of KubeEdge. Once the fix is confirmed, the Security Team will patch the vulnerability in the next patch or minor release, and backport a patch release into the latest three minor releases.
```

Comment:
It would be helpful documenting that we only support latest 3 minor releases here and also in gerneral document about how community manage releases